engine/tester/config_parse.c

```diff
@@ -2940,9 +3107,12 @@ test_value_types_free(test_value_types *types)
 static void
 test_var_arg_free(test_var_arg *p)
 {
-    free(p->name);
-    test_var_arg_values_free(&p->values);
-    free(p->list);
+    if (!p->def_arg)
```

Comment:
I am a bit puzzled by this check: default arguments are never freed?

Reply:
In add_default_arg() I allocate only test_var_arg so here I should free only p. And all default templates are now freed in tester_def_args_free() which is called from tester_global_free().